### ISSUE TITLE
`atmlab` repo is no longer available / tangram installer want Python <3.11 ?

### ISSUE THREAD

**Issue by kx1t** [open-aviation/tangram#1]
Hi, I wanted to experiment with your tangram project, but when installing it, I came across these two issues:

- The `atmlab` project (which is a prereq) is no longer available at https://github.com/xoolive/atmlab -- @xoolive could you advise on what to use instead or how to get it?
- It looks like the `poetry` installer for `tangram` requires Python < 3.11, see the error below. Any reason for this? My default installation uses Python 3.11.2 which is the latest supported version. (This is an easy fix in `pyproject.toml` - I can submit a PR if desired)

Thanks for the advice!

```bash
$ poetry install 
Path /git/atmlab for atmlab does not exist
The currently activated Python version 3.11.2 is not supported by the project (>=3.8,<3.11).
Trying to find and use a compatible version. 

Poetry was unable to find a compatible version. If you have one, you can explicitly use it via the "env use" command.
```

**Comment by xoolive** [comment]
Yes indeed atmlab is a private repo because it accesses private APIs anyway.

We are about to start working on this project again and remove those closed dependencies, etc. and bring the project into a usable state.

Please bear with us in the meantime.

**Comment by xoolive** [comment]
Hi @kx1t things are not perfect yet, but I think the project is usable again.
Things will still keep improving in the coming weeks.
You will find the documentation page here: https://mode-s.org/tangram
